packages/algorithm/package.json

```diff
@@ -26,9 +26,9 @@
     "api": "api-extractor run --local --verbose",
     "build": "npm run build:src && rollup -c",
     "build:src": "tsc --build",
-    "build:test": "tsc --build tests && cd tests && webpack",
+    "build:test": "npm run clean:test && tsc --build tests && cd tests && rollup -c",
     "clean": "rimraf ./lib && rimraf *.tsbuildinfo && rimraf ./types && rimraf ./dist",
-    "clean:test": "rimraf tests/build",
+    "clean:test": "rimraf tests/lib && rimraf tests/tsconfig.tsbuildinfo",
     "docs": "typedoc --options ../../typedoc.js src",
     "minimize": "terser dist/index.js -c -m --source-map \"content='dist/index.js.map',url='index.min.js.map'\" -o dist/index.min.js",
     "test": "npm run test:firefox-headless",
```

Comment:
Should we remove `test:ie`?

Reply:
Yeah, we don't support `ie` anymore :smile:.  Ideally we'd run with [karma-webkit-launcher](https://github.com/google/karma-webkit-launcher) since we claim to support webkit, chrome, and firefox in JupyterLab.